### DISCUSSION TITLE
Upgrade Polaris from v4.0.0 to v4.2.0

### DISCUSSION THREAD

**Discussion by krol3** [aquasecurity/starboard#759] [Development]
Upgrade Polaris from v4.0.0 to v4.2.0

**Comment by danielpacak** [comment]
Upgrading a scanner is a well defined tasks. Unless it requires more effort than bumping up versions we can resolve it by opening a PR directly.

**Comment by krol3** [comment]
Thanks! @danielpacak  I open a PR https://github.com/aquasecurity/starboard/pull/760
